src/situations/accueil/infra/depot_ressources_accueil.js

```diff
@@ -21,13 +28,27 @@ const batiments = {
   tri
 };
 
+const competences = {
+  comprehension_consigne: comprehensionConsigne,
+  rapidite: rapidite,
+  comparaison_tri: comparaisonTri,
+  vigilance_controle: vigilanceControle,
+  organisation_methode: organisationMethode,
+  attention_concentration: attentionConcentration,
+  perseverance: perseverance
+};
+
 export default class DepotRessourcesAccueil extends DepotRessources {
   constructor (chargeurs) {
     super(chargeurs);
-    this.charge([fondAccueil, personnage, precedent, suivant, punaise, casque]);
+    this.charge([fondAccueil, personnage, precedent, suivant, punaise, casque, comprehensionConsigne, rapidite, comparaisonTri, vigilanceControle, organisationMethode, attentionConcentration, perseverance]);
```

Comment:
Ici, plutôt que de répéter encore la liste des compétences, tu peux ajouter une autre ligne : 
this.charge(Object.values(competences));